scripts/duplodocus_local_debug.py

```
@@ -0,0 +1,272 @@
+        str(CFG.ngram_size),
+        "--tokenizer",
+        CFG.tokenizer,
+        "--annotate",
+        "true" if annotate else "false",
+        "--annotate-key",
+        "dedup_info",
+        "--remove-duplicates",
+        "true" if remove_duplicates else "false",
+    ]
+    print("running:", " ".join(cmd))
+    return subprocess.run(cmd, check=True, text=True, capture_output=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)